apps/taxes/src/pages/providers/taxjar/[id]/logs.tsx

```diff
@@ -0,0 +1,45 @@
+import { useRouter } from "next/router";
+import { ClientLogsPage } from "../../../../modules/logs/ui/client-logs-page";
+import { AppPageLayout } from "../../../../modules/ui/app-page-layout";
+import { Section } from "../../../../modules/ui/app-section";
+
+const Header = () => {
+  return <Section.Header>Display logs for your configuration</Section.Header>;
+};
+
+const LogsTaxjarPage = () => {
```

Comment:
this page is the same as "Avatax" page except single breadcrumb label

non blocking but it is also redundant

you can fetch config by ID here, infer if this is taxjar or avatax and display proper routes and breadcrumbs

Reply:
I decided not to block the rendering of the UI if I know what the provider is from the context of the page

Reply:
You dont have to block it, you can render part of the UI, skeletons, event part of breadcrumbs (you know "taxjar" and "avatax" in the URL)